### PR TITLE
Ab#90661 rerender question when trigger activated

### PR DESCRIPTION
**Pull request by MwanPygmay**
# Description

- Add token so can be tested in local => not necessary in real env though
- Refresh the updatedValue when value changed => useful for triggers, otherwise value would not appear.

## Useful links

- [Please insert link to ticket](https://dev.azure.com/WHOHQ/EMSSAFE/_workitems/edit/72584)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

In UAT database, form id 6617d38fb7a3bd5b481d779b, you can test with the regions and countrys dropdowns

## Screenshots

cestlebonbehaviourpointdinterrogation

# Checklist:

( * == Mandatory ) 

- [x] * I have set myself as assignee of the pull request
- [x] * My code follows the style guidelines of this project
- [x] * Linting does not generate new warnings
- [x] * I have performed a self-review of my own code
- [x] * I have put the ticket for review, adding the oort-frontend team to the list of reviewers
- [x] * I have commented my code, particularly in hard-to-understand areas
- [x] * I have put JSDoc comment in all required places
- [x] * My changes generate no new warnings
- [x] * I have included screenshots describing my changes if relevant
- [x] * I have selected labels in the Pull Request, according to the changes with code brings
- [ ] I have made corresponding changes to the documentation ( if required )
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

# More explanation
https://www.loom.com/share/05a716d61b9744faaf51fb304c21d1e5?sid=f87cf896-582a-4f76-93ae-8ceed801b145

### CODE REVIEW

**Comment by AntoineRelief** [comment]
@MwanPygmay 
just to be sure, the token I can remove it right?
Like, comment it

File: libs/shared/src/lib/survey/global-properties/choices-by-graphql.ts [hunk 1]
```diff
@@ -149,28 +186,6 @@ export const render = (questionElement: Question, http: HttpClient): void => {
             }));
           const choiceItems = choices.map((choice) => new ItemValue(choice));
           questionElement.setPropertyValue('visibleChoices', choiceItems);
-          const value = questionElement.value;
```
Comment (AntoineRelief) [inline libs/shared/src/lib/survey/global-properties/choices-by-graphql.ts:152]: @MwanPygmay 
any reason why you removed this code from there?
can't we keep setQuestionValue(..) there as well?

Comment (MwanPygmay) [reply libs/shared/src/lib/survey/global-properties/choices-by-graphql.ts:152]: We can keep it, I just thought that this was included in (questionElement.survey as SurveyModel).onValueChanged but maybe not in all cases

Comment (AntoineRelief) [reply libs/shared/src/lib/survey/global-properties/choices-by-graphql.ts:152]: in that case, i'll bring it back, because it's breaking some logic

**Pull request merged**